### DISCUSSION TITLE
Merge issue when changing case of configmap file

### DISCUSSION THREAD

**Discussion by rmuller-vu** [mia-platform/community#564] [Q&A]
### What product are you having troubles with?

Console

### What Console version are you using?

1.102.8

### Description

This is related to [discussion 563](https://github.com/mia-platform/community/discussions/563). If we rename a file to change the casing as specified in discussion 563 we see an error when merging into a different branch that has the wrong casing. 

revision `main` has configmap file named `casing-test.json` and revision `update-casing` has a configmap file named `Casing-test.json`. When we try to merge `update-casing` into `main` we see an error.

The merge window shows both files. One shows message saying `The file will be removed` then other shows `The file will be added`.
![image](https://github.com/user-attachments/assets/ffa6d4d1-441a-476c-a594-2579c3823d5a)

Below is the error we receive when clicking confirm merge
![image](https://github.com/user-attachments/assets/c8f40332-631b-4c31-b0a5-aae876096efa)

**Comment by marcofilippi** [comment]
Hi @rmuller-vu 

Thanks again for pointing this out. 

As mentioned in the [discussion 563](https://github.com/mia-platform/community/discussions/563), we've tracked this issue and will dig into it. 

In the meantime, your Mia-Platform referent will keep you updated on the progress.
